guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md

```diff
@@ -39,46 +38,48 @@ This command has no effect on cron jobs outside the `#~ MAGENTO START` and `#~ M
 
 Command options:
 
+```bash
   bin/magento cron:run [--group="<cron group name>"]
+```
 
 where `--group` specifies the cron group to run (omit this option to run cron for all groups)
 
 To run the indexing cron job, enter:
 
 `php bin/magento cron:run --group index`
 
-
 To run the default cron job, enter:
 
 `php bin/magento cron:run --group default`
 
-
 To set up custom cron jobs and groups, see [Configure custom cron jobs and cron groups]({{ page.baseurl }}/config-guide/cron/custom-cron.html).
 
 {:.bs-callout .bs-callout-info}
 You must run cron twice: the first time to discover tasks to run and the second time — to run the tasks themselves. The second cron run must occur on or after the `scheduled_at` time for every task.
 
 ## Logging
 
-System logs provide detailed information for debugging. The following attributes apply to Magento cron logs:
+With Magento 2.3.1, `cron` job information is moved from `system.log` and into a separate `cron.log`.
+By default, the cron information can be found at `<install_directory>/var/log/cron.log`. 
+All exceptions from cron jobs are logged by `\Magento\Cron\Observer\ProcessCronQueueObserver::execute`.
 
--   All exceptions from cron jobs are logged by `\Magento\Cron\Observer\ProcessCronQueueObserver::execute`.
+In addition to being logged in `cron.log`:
 
--   Failed jobs with `ERROR` and `MISSED` statuses are logged to the `<your Magento install dir>/var/log` directory.
+- Failed jobs with `ERROR` and `MISSED` statuses are logged to the `<install_directory>/var/log` directory.
 
--   Jobs with an `ERROR` status are always logged as `CRITICAL` in `<your Magento install dir>/var/log/exception.log`.
+- Jobs with an `ERROR` status are always logged as `CRITICAL` in `<install_directory>/var/log/exception.log`.
 
--   Jobs with a `MISSED` status are logged as `INFO` in the `<your Magento install dir>/var/log` directory (developer mode only).
+- Jobs with a `MISSED` status are logged as `INFO` in the `<install_directory>/var/log` directory (developer mode only).
```

Comment:
Consider putting these 3 bullet items into a table (not sure if there's enough info at the moment)

Reply:
Don't see it as tabular data. A bit too verbose I think.